jdaviz/container.vue

```diff
@@ -19,7 +19,7 @@
     <gl-component
       v-for="(viewer, index) in stack.viewers"
       :key="viewer.id"
-      :title="viewer.id"
+      :title="viewer.config === 'cubeviz' ? viewer.name : viewer.id"
```

Comment:
would we prefer `viewer.name` ("Uncertainty") or `viewer.reference` ('uncert-viewer')?

Reply:
I don't know the implications of that change for Imviz, so I don't really want to mess with it right now.

Reply:
Yeah, I just checked and that breaks Imviz viewer creation.

Reply:
Oh, I misread your comment, I thought you meant change viewer.id to viewer.reference - reference is probably better than name, I agree. I'll change that, one sec.